cabal-plan.cabal

```diff
@@ -1,5 +1,5 @@
 name:                cabal-plan
-version:             0.1.1.0
+version:             0.1.1.1
```

Comment:
This is merely a technicality note (no need to change anything): you called this  "bump minor version" whereas it's rather a patch-level increment (which is enough, as the exposed API of the library didn't change).